### ISSUE TITLE
Bump to tabtab@2

### ISSUE THREAD

**Issue by sindresorhus** [yeoman/yo#508]
Tried doing it, but tests were [failing on Travis](https://travis-ci.org/yeoman/yo/jobs/200700423), but not locally.

// @mklabs

**Comment by mklabs** [comment]
Hi Sindre,

I need to either rework on yeoman or remove tabtab from yeoman core, but can't do so at the moment.

With all due respect, 
Mickael

**Comment by mklabs** [comment]
@sindresorhus no promise here, but I'll try to take a look. Do you have any active pull request for these changes ?

**Comment by sindresorhus** [comment]
No

**Comment by mklabs** [comment]
@sindresorhus Do you still need help ?

**Comment by sindresorhus** [comment]
@mklabs Yes. I didn't upgrade `tabtab` in those commits as I had problems last time I tried to update it. Don't remember exactly what though.

**Comment by mklabs** [comment]
Okay, we need to talk. What is your prefered way ? Hangout ?

**Comment by mklabs** [comment]
@sindresorhus I can discord, hangout and skype

**Comment by sindresorhus** [comment]
I’m on hangout

**Comment by mklabs** [comment]
Let's do this, not right now tho and I need to configure this with my headset, maybe in 2 hours, are you okay with this ?

**Comment by sindresorhus** [comment]
I meant hangouts text chat. I can’t do voice right now.

**Comment by mklabs** [comment]
oh ok 👍

**Comment by MikeMcC399** [comment]
- This was resolved through PR https://github.com/yeoman/yo/pull/915 and should be closed
